### PR TITLE
matrix env for mpi testing

### PR DESCRIPTION
**Pull request by alperyeg**
This PR adds an additional environment to the matrix in `travis.yml` and corresponding changes in `install.sh`. With this env we should be able to test for mpi via `mpi4py`.

### CODE REVIEW

File: continuous_integration/test_script.sh [hunk 1]
```diff
@@ -13,7 +13,11 @@ python -c "import numpy; print('numpy %s' % numpy.__version__)"
 python -c "import scipy; print('scipy %s' % scipy.__version__)"
 
 if [[ "$COVERAGE" == "true" ]]; then
-    nosetests --with-coverage --cover-package=elephant
+    if [["$MPI == true" ]]; then
+	mpiexec -n 2 nosetests --with-coverage --cover-package=elephant
```
Comment (mdenker) [inline continuous_integration/test_script.sh]: I guess this should be 
`"$MPI" == "true"`
to work...

File: continuous_integration/test_script.sh [hunk 2]
```diff
@@ -13,7 +13,7 @@ python -c "import numpy; print('numpy %s' % numpy.__version__)"
 python -c "import scipy; print('scipy %s' % scipy.__version__)"
 
 if [[ "$COVERAGE" == "true" ]]; then
-    if [["$MPI == "true" ]]; then
+    if [["$MPI" == "true" ]]; then
```
Comment (mdenker) [inline continuous_integration/test_script.sh]: Still throwing an error, I think it needs a space after [[:
`[[ "$MPI" ...`

Comment (alperyeg) [reply continuous_integration/test_script.sh]: Are you testing locally?

Comment (mdenker) [reply continuous_integration/test_script.sh]: No, but travis had errors on that bash line such that mpiexec was not executed. Now it seems it is, and creates "real" errors...

**Comment by mdenker** [comment]
This PR sets up the environment for parallel tests, but individual parallel codes must still be adapted to make use of `mpirun n>1`.

**Pull request merged**